server/db/models/user.js

```diff
@@ -28,6 +51,7 @@ const User = db.define('user', {
       return () => this.getDataValue('salt')
     }
   },
+  // not sure if googleId is needed in seed data was part of boiler plate
```

Comment:
I think we'll need this later when we implement logging in with facebook and google and such so maybe leave it in for now?

Reply:
sounds good